### ISSUE TITLE
Jabba changes seems to be ignored on Mac OS Catalina with ZSH shell

### ISSUE THREAD

**Issue by exelanz** [shyiko/jabba#631]
Hi, I installed Jabba. Version 0.11.2. Installation through homebrew. Jabba commands work fine, no errors, however, no changes are made. I can install java versions, but when using "Jabba use java version" nothing happens. Running "Jabba current" displays nothing.

**Comment by akloeber** [comment]
Same here. No errors but `$PATH` and `$JAVA_HOME` remain unchanged und `jabba current` yields nothing.

**Comment by akshaydewan** [comment]
I noticed that there isn't any `~/.jabba/jabba.sh` file present when installed from Homebrew. So sourcing `[ -s "$JABBA_HOME/jabba.sh" ] && source "$JABBA_HOME/jabba.sh"` wasn't working. I couldn't find this `.sh` under `Cellar` either.

I ended up installing it without homebrew and it works fine.

**Comment by michaelcusick** [comment]
Yes, I ran into the same issue and can confirm that @akshaydewan's solution works. I'll look into fixing the Homebrew package.
